.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dockerFile": "Dockerfile",
+  "extensions": [
+    "pgourlain.erlang"
```

Comment:
What is this extension?

Reply:
It's an extension we've used at work, it provides some very basic language support. Syntax highlighting, and some auto-complete.